### DISCUSSION TITLE
Feat：请支持 ttu-ttu

### DISCUSSION THREAD

**Discussion by myxiaoao** [tw93/Pake#106] [General]
Epub 阅读器 ttu-ttu，它是一个开源的网页应用，可以离线使用，相当于你部署一个自己的“微信读书”。

[https://reader.ttsu.app](https://reader.ttsu.app)

Github: https://github.com/ttu-ttu/ebook-reader

感谢开发者们。

**Comment by tw93** [comment]
试了试可以用，你去玩玩
[Reader_1.0.0_universal.dmg.zip](https://github.com/tw93/Pake/files/10090465/Reader_1.0.0_universal.dmg.zip)

![image](https://user-images.githubusercontent.com/8736212/203950327-e7dab8b7-ae71-476f-82fe-7f8b64aa4c55.png)

**Comment by myxiaoao** [comment]
试了下，不错！
